### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by Kodylow**
- Updates readme with clovyr links for guardian, gateway, and dev environment.
- Rewrites intro and adds relevant links out to developer resources
- Puts dev env setup instructions on this page

For our remaining docs I want to migrate those to fedimint.org or the wiki, I've been trying to come up with good ways to reorganize within docs but it's always clunky to navigate.

### CODE REVIEW

**Comment by justinmoon** [comment]
@EthnTuttle

File: README.md [hunk 1]
```diff
@@ -20,15 +19,76 @@
...
+[Fedimint](https://github.com/fedimint/fedimint) is a module based system for building federated applications. It is designed to be a trust-minimized, censorship-resistant, and private alternative to centralized applications.
+
+Fedimint ships with 3 default modules: Bitcoin, Lightning, and Chaumian Ecash, for out-of-the-box best practices for private and trust-minimized payments. [You can write custom modules](https://github.com/fedimint/fedimint-custom-modules-example) that define further consensus items and transaction types leveraging the payments modules to build your own federated applications.
+
+The Fedimint Developer Discord is the best place to get help and ask questions. [Join the Discord](https://discord.gg/cEVEmqCgWG) and say hi! We are extremely active and work to onboard developers or all skill levels to Fedimint and associated open-source Bitcoin projects. Fedimint touches many different areas of Bitcoin development, so there is something for everyone. See below for more information on how to get involved.
```
Comment (EthnTuttle) [inline README.md:26]:
```suggestion
The Fedimint Developer Discord is the best place to get help and ask questions. [Join the Discord](https://discord.gg/cEVEmqCgWG) and say hi! We are extremely active and work to onboard developers of all skill levels to Fedimint and associated open-source Bitcoin projects. Fedimint touches many different areas of Bitcoin development, so there is something for everyone. See below for more information on how to get involved.
```

File: README.md [hunk 2]
```diff
@@ -20,15 +19,76 @@
...
+
+Fedimint is alpha software and recently cut its 0.1 release. We recommend you run Fedimint on testnets like mutinynet, or on mainnet with small amounts of money.
```
Comment (EthnTuttle) [inline README.md:28]: Should leave with this and make it BOLD.

File: README.md [hunk 3]
```diff
@@ -20,15 +19,76 @@
...
+
+## Running your own Fedimint
+
+It's easy to set up and run your own federations. Fedimint is designed to be Byzantine fault tolerant so is resilient to `m` malicious nodes in a federation of `3m + 1` nodes. If you run a federation of 4 guardians you are malicious to 1 guardian, if you run a federation of 7 guardians you are malicious to 2 guardians, etc. However Fedimint can also be run with smaller configurations (1/1, 2/2, 2/3) for testing and development.
```
Comment (EthnTuttle) [inline README.md:32]: This needs some clarification. Especially the last sentence.

File: README.md [hunk 4]
```diff
@@ -20,15 +19,76 @@
...
+
+To do lightning payments, Fedimint requires a Lightning Gateway: a user of the federation that accepts ecash in exchange for sending/receiving lightning payments. The Lightning Gateway is not a guardian and acts as an untrusted economic actor serving the federation.
```
Comment (EthnTuttle) [inline README.md:34]:
```suggestion
To handle lightning payments, Fedimint requires a Lightning Gateway: a user of the federation that accepts ecash in exchange for sending/receiving lightning payments. The Lightning Gateway is not a guardian and acts as an untrusted economic actor serving the federation.
```

File: README.md [hunk 5]
```diff
@@ -20,15 +19,76 @@
...
+
+### Running Fedimint on Mutinynet
+
+The easiest way to run Fedimint on the Mutinynet testnet is with Clovyr, a decentralized application platform. Click the Launch buttons below to spin up a Fedimint guardian or Lightning Gateway in a couple minutes. Clovyr handles hosting, DNS, and configuration for you.
```
Comment (EthnTuttle) [inline README.md:38]: It's way longer than a couple of minutes. 😆

File: README.md [hunk 6]
```diff
@@ -20,15 +19,76 @@
...
+
+- [The Fedimint Developer Discord](https://discord.gg/cEVEmqCgWG): the best place to get help and ask questions.
+- [Fedimint Contributor Calendar](https://calendar.google.com/calendar/u/0/embed?src=fedimintcalendar@gmail.com): This calendar contains all the developer calls and events.
+- [Fedimint Developer Calls](https://meet.jit.si/fedimintdevcall): We have developer calls every monday at 4PM UTC to review PRs and discuss current development priorities. As a new developer, this is a great place to find good first issues and mentorship from the core team on how to get started contributing to Fedimint.
+- [Weekly Deep Dive](https://meet.jit.si/fedimintdevcall): We have a deep dive every thursday at 4PM UTC to discuss technical topics relating to fedimint in depth: cryptography, rust programming, consensus, networking, etc. This is a great place to learn about the internals of Fedimint and Bitcoin. We normally plan these calls based off requests from contributors on aspects of Fedimint they want to learn more about, so please reach out if you have a topic you want to learn more about.
```
Comment (EthnTuttle) [inline README.md:52]: Should capitalize Fedimint here if we do it elsewhere. And Rust.

File: README.md [hunk 7]
```diff
@@ -20,15 +19,76 @@
...
+- [UI/UX Design Calls](https://meet.jit.si/fedimintdevcall): We have UI/UX design calls every wednesday at 4PM UTC to discuss the UI/UX of Fedimint and associated projects. This is a great place to get involved if you are a UI/UX designer or developer with React/Typescript experience.
+
+### Fedimint Repos and Projects to Contribute To
+
+- [Fedimint](https://github.com/fedimint/fedimint): The core Fedimint repository. This is where the core consensus and networking code lives. Fedimint is an advanced Rust project and is a great place to learn Rust, cryptography, networking, consensus, and bitcoin development. We have a lot of good first issues, are happy to mentor new developers, and are always looking for experienced Rust developers to help with the core codebase.
```
Comment (EthnTuttle) [inline README.md:57]: Link to `good first issues` here. And maybe above in the "For Developers" section.

File: README.md [hunk 8]
```diff
@@ -20,15 +19,76 @@
...
+- [UI](https://github.com/fedimint/ui): The Fedimint Guardian and Lightning Gateway UIs. These are written in Typescript and React and always need help with UI/UX design and development to make Fedimint more user friendly.
```
Comment (EthnTuttle) [inline README.md:58]:
```suggestion
- [UI](https://github.com/fedimint/ui): The default Fedimint Guardian and Lightning Gateway UIs. These are Typescript and React project. Contributing to this repo helps with UI/UX design and code to make Fedimint more user friendly.
```

**Review by EthnTuttle — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (Kodylow) [reply README.md:38]: just to launch with clovyr is a couple mins, especially just the guardians part. keeping it.

Comment (EthnTuttle) [reply README.md:40]: 🫡

File: README.md [hunk 9]
```diff
@@ -19,19 +19,21 @@
   <img alt="Lines of code" src="https://img.shields.io/tokei/lines/github/fedimint/fedimint">
 </p>
 
-[Fedimint](https://github.com/fedimint/fedimint) is a module based system for building federated applications. It is designed to be a trust-minimized, censorship-resistant, and private alternative to centralized applications.
+[Fedimint](https://fedimint.org) is a module based system for building federated applications. It is designed to be a trust-minimized, censorship-resistant, and private alternative to centralized applications.
 
-Fedimint ships with 3 default modules: Bitcoin, Lightning, and Chaumian Ecash, for out-of-the-box best practices for private and trust-minimized payments. [You can write custom modules](https://github.com/fedimint/fedimint-custom-modules-example) that define further consensus items and transaction types leveraging the payments modules to build your own federated applications.
+> **Fedimint is alpha software released under an [MIT License](https://github.com/fedimint/fedimint/blob/master/LICENSE) and recently cut its 0.1 release. We recommend you run Fedimint on testnets like mutinynet, or on mainnet with small amounts of money.**
```
Comment (EthnTuttle) [inline README.md:24]: Perhaps we should recommend users check out the latest release (include link to releases) so this sentence can stay evergreen?

File: README.md [hunk 10]
```diff
@@ -21,7 +21,7 @@
 
 [Fedimint](https://fedimint.org) is a module based system for building federated applications. It is designed to be a trust-minimized, censorship-resistant, and private alternative to centralized applications.
 
-> **Fedimint is alpha software released under an [MIT License](https://github.com/fedimint/fedimint/blob/master/LICENSE) and recently cut its 0.1 release. We recommend you run Fedimint on testnets like mutinynet, or on mainnet with small amounts of money.**
+> **Fedimint is alpha software released under an [MIT License](https://github.com/fedimint/fedimint/blob/master/LICENSE) and recently cut its [0.1 release](https://github.com/fedimint/fedimint/releases/tag/v0.1.3). We recommend you run Fedimint on testnets like mutinynet, or on mainnet with small amounts of money.**
```
Comment (elsirion) [inline README.md:24]: You could just link to https://github.com/fedimint/fedimint/releases/latest, so the most recent release is always behind this link

Comment (Kodylow) [reply README.md]: updated

**Pull request merged**